core/common/src/main/java/alluxio/security/authorization/AccessControlList.java

```diff
@@ -88,6 +90,53 @@ public short getMode() {
     return new Mode(owner, group, other).toShort();
   }
 
+  /**
+   * @return an immutable list of ACL entries
+   */
+  public List<AclEntry> getEntries() {
+    ImmutableList.Builder<AclEntry> builder =
+        new ImmutableList.Builder<>();
+    builder.add(new AclEntry.Builder()
+        .setType(AclEntryType.OWNING_USER)
+        .setSubject(mOwningUser)
+        .setActions(getOwningUserActions())
+        .build());
+    for (Map.Entry<String, AclActions> kv : mUserActions.entrySet()) {
+      if (kv.getKey().equals(OWNING_USER_KEY)) {
+        continue;
```

Comment:
Instead of continue, could we put the above logic here?

Reply:
fixed